data/templates/hipster/config.yml

```diff
@@ -0,0 +1,8 @@
+name: Hipster Barista
+default:
+-
+-
+link: http://knowyourmeme.com/memes/hipster-barista
+aliases:
+- hipster-barista
+- barista
```

Comment:
I'm not sure it's a great idea to have short 2 letter abbreviations.  They might collide in the future, or simply be unrecognizable.